keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -127,27 +138,47 @@ def text_to_image(
             )
         phrase = inputs + [49407] * (MAX_PROMPT_LENGTH - len(inputs))
         phrase = tf.convert_to_tensor([phrase], dtype=tf.int32)
-        phrase = tf.repeat(phrase, batch_size, axis=0)
 
-        # Encode prompt tokens + positions into a "context" vector
-        pos_ids = tf.convert_to_tensor([list(range(MAX_PROMPT_LENGTH))], dtype=tf.int32)
-        pos_ids = tf.repeat(pos_ids, batch_size, axis=0)
-        context = self.text_encoder.predict_on_batch([phrase, pos_ids])
+        context = self.text_encoder.predict_on_batch([phrase, self._get_pos_ids()])
 
-        # Encode unconditional tokens + positions as "unconditional context"
-        unconditional_tokens = tf.convert_to_tensor(
-            [_UNCONDITIONAL_TOKENS], dtype=tf.int32
-        )
-        self.unconditional_tokens = tf.repeat(unconditional_tokens, batch_size, axis=0)
-        unconditional_context = self.text_encoder.predict_on_batch(
-            [self.unconditional_tokens, pos_ids]
+        return context
+
+    def generate_image(
```

Comment:
I think they should be exposed to users so that users can easily do walks through the encoder latent manifold and diffusion noise space (will send out tutorial on this soon)